src/components/pages/AdminHome/AdminCourseCard.js

```diff
@@ -20,19 +19,28 @@ export default function CourseCard(props) {
     <div className="admin-course-card-container">
       <Card title={course.course_name}>
         <h3>
-          {headers[17].toUpperCase()}{' '}
-          <span className="black">{course.program_name}</span>
+          Program:
+          <span className="black">{' ' + course.program_name}</span>
         </h3>
         {/*<h3>
```

Comment:
Make sure to check if this comment is still relevant. It looks like the headers array is being phased out so the instruction left behind may no longer be necessary